index.js

```diff
@@ -269,6 +272,18 @@ function init (ssb, config) {
           })
         })
       },
+      get: tribeGet,
+      // get (id, cb) {
```

Comment:
confirm with @mixmix  first about whether we need to return the dmKey as well

Reply:
leave that to me - I need to put a PR in which deals with dmKeys